requirements-latest.txt

```diff
@@ -72,7 +73,8 @@ validate-email>=1.3
 vmprof>=0.4.15
 watchdog>=0.8.3
 web3>=5.13.1
-werkzeug>=0.14.1
+# ImportError: cannot import name 'escape' from 'werkzeug.utils'
+werkzeug>=0.14.1,<2
```

Comment:
Why `<2`? `werkzeug.utils.escape` was removed in 2.1.0, see [changelog](https://werkzeug.palletsprojects.com/en/2.1.x/changes/#version-2-1-0).

Reply:
yeah, maybe <2.1 would work as well. not sure. eg flask needs werkzeug etc., but I didn't test it exhaustively to figure it out https://gist.github.com/oberstet/726787d3acb0da736ee194c0e3af914f

Reply:
also, this should be looked at together with https://github.com/crossbario/crossbar/blob/ecc8b4e0d9f8fa3efa7d59755cb2e16151055870/requirements-min.txt#L32 .. at least I ran into both at the same time recently ..